tests/utils/test_utils.py

```diff
@@ -42,6 +42,7 @@ def test_issue_parsing(self):
         self.assertEqual(parsed_result.alternatives, "tititata")
         self.assertEqual(parsed_result.licence, "Apache License 2.0")
         self.assertEqual(parsed_result.affiliations, None)
+        self.assertEqual(parsed_result.ibm_maintained, True)
```

Comment:
If there are two test issues, why is only one assertion added?

Reply:
For some reason, `test_utils` only runs on one test issue. The other issue is used in `test_cli`.